### PR TITLE
docs: `Tutorials` update

### PR DESCRIPTION
**Pull request by leo-gan**
A big update of the `Tutorials` page. Cleaned it up. Added several new resources.

### CODE REVIEW

File: docs/docs/additional_resources/tutorials.mdx [hunk 1]
```diff
@@ -1,155 +1,55 @@
 # Tutorials
 
-Below are links to tutorials and courses on LangChain. For written guides on common use cases for LangChain, check out the [use cases guides](/docs/use_cases).
+### Different materials
```
Comment (eyurtsev) [inline docs/docs/additional_resources/tutorials.mdx:3]: Could we keep a link to the our own internal Use Cases docs at the top level?

The Use Case docs are more likely to stay up to date as the framework evolves and will likely be expanded to cover more of the nuance of the use cases.

Comment (eyurtsev) [inline docs/docs/additional_resources/tutorials.mdx:3]: `Different Materials` isn't super descriptive as a name for me. Should probably be `##`

File: docs/docs/additional_resources/tutorials.mdx [hunk 2]
```diff
@@ -1,155 +1,55 @@
...
 
-### [LangChain](https://en.wikipedia.org/wiki/LangChain) on Wikipedia
+- [LangChain AI Handbook](https://www.pinecone.io/learn/langchain/) By **James Briggs** and **Francisco Ingham**
+- [LangChain Cheatsheet](https://pub.towardsai.net/langchain-cheatsheet-all-secrets-on-a-single-page-8be26b721cde) by **Ivan Reznikov**
 
-### Books
+#### [LangChain documentation: Use cases](/docs/use_cases)
 
-#### [Generative AI with LangChain](https://www.amazon.com/Generative-AI-LangChain-language-ChatGPT/dp/1835083463/ref=sr_1_1?crid=1GMOMH0G7GLR&keywords=generative+ai+with+langchain&qid=1703247181&sprefix=%2Caps%2C298&sr=8-1) by [Ben Auffrath](https://www.amazon.com/stores/Ben-Auffarth/author/B08JQKSZ7D?ref=ap_rdr&store_ref=ap_rdr&isDramIntegrated=true&shoppingPortalEnabled=true), ©️ 2023 Packt Publishing
+#### [LangChain](https://en.wikipedia.org/wiki/LangChain) in Wikipedia
```
Comment (eyurtsev) [inline docs/docs/additional_resources/tutorials.mdx:16]: Don't think this belongs here

```suggestion
```

File: docs/docs/additional_resources/tutorials.mdx [hunk 3]
```diff
@@ -1,155 +1,55 @@
...
-- [`GPT-4` Tutorial: How to Chat With Multiple `PDF` Files (~1000 pages of Tesla's 10-K Annual Reports)](https://youtu.be/Ix9WIZpArm0)
-- [`GPT-4` & LangChain Tutorial: How to Chat With A 56-Page `PDF` Document (w/`Pinecone`)](https://youtu.be/ih9PBGVVOO4)
-- [LangChain & `Supabase` Tutorial: How to Build a ChatGPT Chatbot For Your Website](https://youtu.be/R2FMzcsmQY8)
-- [LangChain Agents: Build Personal Assistants For Your Data (Q&A with Harrison Chase and Mayo Oshin)](https://youtu.be/gVkF8cwfBLI)
-
-
-### Codebase Analysis
-- [Codebase Analysis: Langchain Agents](https://carbonated-yacht-2c5.notion.site/Codebase-Analysis-Langchain-Agents-0b0587acd50647ca88aaae7cff5df1f2)
 
+- [by Nicholas Renotte](https://youtu.be/MlK6SIjcjE8)
```
Comment (eyurtsev) [inline docs/docs/additional_resources/tutorials.mdx:48]: hmm this is easier to keep in sync, but much less obvious what the content is. 

Users might benefit from being able to see the contents of the lectures? Not sure. We could do a mixture of the two approaches and link to play list but also to some individual tutorials?

The main problem is that the tutorials might fall out of date...

cc @baskaryan what do you think?

File: docs/docs/additional_resources/tutorials.mdx (hunk 2 above)
Comment (leo-gan) [reply docs/docs/additional_resources/tutorials.mdx:16]: yes. It is definitely not Tutorials.

File: docs/docs/additional_resources/tutorials.mdx (hunk 1 above)
Comment (leo-gan) [reply docs/docs/additional_resources/tutorials.mdx:3]: @eyurtsev  I'd like also to see something more descriptive. Do you have something on your mind? I need several options.

File: docs/docs/additional_resources/tutorials.mdx (hunk 3 above)
Comment (leo-gan) [reply docs/docs/additional_resources/tutorials.mdx:48]: I've removed all lines with lecture/video names. The page looks "dirty" with them. But I checked on the built page and now it is more clean, readable, and "navigable". It is better now for sure.
BTW I sorted tutorials by quality.

File: docs/docs/additional_resources/tutorials.mdx (hunk 1 above)
Comment (leo-gan) [reply docs/docs/additional_resources/tutorials.mdx:3]: After second thought... Use Cases are NOT good as Tutorials. There are many pages, like Get started / Quickstart, etc. but honestly nothing looks like tutorial. If we need the "Official" tutorial, it is TODO for now. So, U promoted Use Cases to the top level, but moved it to back, after Tutorials and Courses.

**Review by baskaryan — APPROVED** [review]

**Comment by leo-gan** [comment]
@baskaryan Thanks!

**Pull request merged**